src/style.css

```diff
@@ -0,0 +1,83 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.form {
+  display: flex;
+  width: 100%;
+  height: 70px;
+  margin-top: 20px;
+}
+
+.form-small {
+  height: 42px;
+}
+
+.label {
+  display: block;
+  position: relative;
+  width: 100%;
+}
+
+.label::before {
+  position: absolute;
+  content: "";
+  height: 19px;
+  width: 19px;
+  top: 25px;
+  left: 26px;
+  background-image: url(images/Search.svg);
+  background-size: contain;
+  background-repeat: no-repeat;
```

Comment:
Combine with `background` shorthand property

Reply:
What does it mean?